sdk/cognitiveservices/azure-cognitiveservices-inkrecognizer/requirements.txt

```diff
@@ -0,0 +1,3 @@
+six>=1.6
```

Comment:
pleas change this file to a dev_requirements that contains only your dev dependencies. That should look like this:
https://github.com/Azure/azure-sdk-for-python/blob/master/sdk/cognitiveservices/azure-cognitiveservices-language-textanalytics/dev_requirements.txt

Reply:
I'm not familiar with this. Is the content in the sample, i.e.
https://github.com/Azure/azure-sdk-for-python/blob/ec7eef4d2e57da01a61f793b191f9fe9613d8741/sdk/cognitiveservices/azure-cognitiveservices-language-textanalytics/dev_requirements.txt#L1-L2

everything I need to add to my dev_requirements.txt? Or I should edit based on my need? I know I need to use azure-sdk-tools to adapt test framework on my test code, but not sure about azure-mgmt-cognitiveservices.

Reply:
Please add azure-core as well, see comment on:
https://github.com/Azure/azure-sdk-for-python/pull/7247#discussion_r329259513